src/chainparams.cpp

```diff
@@ -243,6 +243,9 @@ class CMainParams : public CChainParams {
         consensus.accountDestruction.insert(GetScriptForDestination(DecodeDestination("dJEbxbfufyPF14SC93yxiquECEfq4YSd9L", *this)));
         consensus.accountDestruction.insert(GetScriptForDestination(DecodeDestination("8UAhRuUFCyFUHEPD7qvtj8Zy2HxF5HH5nb", *this)));
 
+        consensus.smartContracts.clear();
+        consensus.smartContracts["DFIP2201"] = GetScriptForDestination(CTxDestination(WitnessV0KeyHash(std::vector<unsigned char>{0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0})));
```

Comment:
Can we use `std::vector<unsigned char>(20, 0)` `{}` - it's `initializer_list`

Reply:
For this leave it in full, next smart contract address will be copy and paste of this with 1 set at the end.

Reply:
May be we use something like a boost preprocessor macro to update in compile time and let that increment later. For now copy paste looks to do.